src/components/CodeSnippet/CodeSnippet.test.tsx

```diff
@@ -44,26 +45,31 @@ describe("CodeSnippet ", () => {
         ]}
       />
     );
-    const lines = wrapper.find(
-      ".p-code-snippet__block--numbered .p-code-snippet__line"
-    );
-    expect(lines.length).toBe(3);
-    expect(lines.first().text()).toEqual("Test line 1;");
+    expect(
+      // eslint-disable-next-line testing-library/no-node-access
+      document.querySelector(".p-code-snippet__block--numbered")
+    ).toBeInTheDocument();
+    expect(screen.getByText("Test line 1;")).toBeInTheDocument();
+    expect(screen.getByText("Test line 2;")).toBeInTheDocument();
+    expect(screen.getByText("Test line 3;")).toBeInTheDocument();
   });
 
   it("renders default linux prompt icon", () => {
-    const wrapper = mount(
+    render(
       <CodeSnippet
         blocks={[
           { appearance: CodeSnippetBlockAppearance.LINUX_PROMPT, code: "Test" },
         ]}
       />
     );
-    expect(wrapper.find(".p-code-snippet__block--icon").length).toBe(1);
+    expect(
```

Comment:
Could we test it other way around - get element by text (Test) and check if it has a classname with icon suffix?

Reply:
Same for other similar tests in same file.

Reply:
The node that contains the text is in a child of the node with the class:
```html
<pre class="p-code-snippet__block--icon">
  <code>Test</code>
</pre>
```

I guess another way of doing this would be to use `.closest()` to find the parent element with the class (e.g. `screen.getByText("Test").closest(".p-code-snippet__block--icon")`) but the linter will also complain about that as `closest` is a standard DOM function, not a part of RTL.

Reply:
Oh, that's fair, let's keep it for now.